.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
should this be here? It looks like an editor specific file. Could you also add it to .gitignore?

Reply:
ah. I see @Aprillion wants this - disregard this comment :D